client/app/billing/autoRenew/actions/update/automatic/automatic.component.js

```diff
@@ -0,0 +1,10 @@
+import template from './automatic.html';
+
+export default {
+  bindings: {
+    paymentMean: '<',
```

Comment:
```suggestion
    meansOfPayment: '<',
```
?
😄

Reply:
I prefer the other way round :) 
Also name is quite short so I thing the `of` would add extra length but readability is equal